config/nginx.conf

```diff
@@ -0,0 +1,162 @@
+# Set another default user than root for security reasons
```

Comment:
@nlMeminger The project template provides a nginx config file. Might be useful for you to look through.